### PR TITLE
Support part distribute

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

File: src/storage/admin/StatisTask.cpp [hunk 1]
```diff
@@ -170,18 +186,27 @@ StatisTask::genSubTask(GraphSpaceID spaceId,
...
                 lastEdgeType  = edgeType;
                 lastRank = ranking;
                 lastDstVertexId = destination;
+
+                uint64_t vid = 0;
+                if (isIntId) {
+                    memcpy(static_cast<void*>(&vid), destination.data(), 8);
+                } else {
+                    nebula::MurmurHash2 hash;
+                    vid = hash(destination.data());
```
Comment (panda-sheep) [inline src/storage/admin/StatisTask.cpp:205]: It is best to use `toString `here,  refer https://github.com/vesoft-inc/nebula-storage/pull/248

**Review by panda-sheep — COMMENTED** [review]
Good job!

File: src/storage/admin/StatisTask.cpp (hunk 1 above)
Comment (panda-sheep) [reply src/storage/admin/StatisTask.cpp]: line 187, I has convertted

**Review by panda-sheep — DISMISSED** [review]
Good job!

**Comment by liuyu85cn** [comment]
may be need to merge result from task, like what StatisJobExecutor::finish did?

**Review by panda-sheep — APPROVED** [review]
LGTM

**Review by panda-sheep — APPROVED** [review]

**Review by bright-starry-sky — APPROVED** [review]
LGTM

**Pull request merged**